health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/hooks/useUpdateAppConfig.js

```
@@ -33,7 +33,7 @@ const updateAppConfigService = async (req, tenantId) => {
         });
     return response;
   } catch (error) {
-    throw new Error(error?.response?.data?.Errors?.[0].code);
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)